pyresample/utils.py

```diff
@@ -407,6 +407,34 @@ def proj4_str_to_dict(proj4_str):
     return dict((x[0], (x[1] if len(x) == 2 else True)) for x in pairs)
 
 
+def proj4_radius_parameters(proj4_dict_or_str):
+    """Calculate 'a' and 'b' radius parameters.
+
+    Returns:
+        a (float), b (float): equatorial and polar radius
+    """
+    if isinstance(proj4_dict_or_str, str):
+        new_info = proj4_str_to_dict(proj4_dict_or_str)
```

Comment:
I find the names here a bit confusing, can either proj4_str_to_dict or proj4_dict_or_str be renamed ?